PLUB/Configuration/Extensions/Ex+UIImage.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Ex+UIImage.swift
+//  PLUB
+//
+//  Created by 김수빈 on 2023/03/26.
+//
+
+import UIKit
+
+extension UIImage {
+  
+  public convenience init?(
```

Comment:
public은 지워주시면 될 것 같습니다.

```suggestion
 convenience init?(
```